src/components/atom_preference_language_checkbox/index.tsx

```diff
@@ -1,44 +1,21 @@
 import { GlobalLanguageCode } from '@/global.interface'
 import { preferenceState } from '@/recoil/preferences/preference.state'
 import { Checkbox, FormControlLabel } from '@mui/material'
-import { FC, SyntheticEvent } from 'react'
-import { useRecoilCallback, useRecoilValue } from 'recoil'
+import { FC } from 'react'
+import { useRecoilValue } from 'recoil'
 
 interface Props {
   languageCode: GlobalLanguageCode
 }
 
 const PreferenceLanguageCheckbox: FC<Props> = ({ languageCode }) => {
   const preference = useRecoilValue(preferenceState)
-  const onChange = useRecoilCallback(
-    ({ set, snapshot }) =>
-      async (e: SyntheticEvent<Element, Event>, checked: boolean) => {
-        const preference = await snapshot.getPromise(preferenceState)
-        if (!preference) return
-
-        const nativeLanguagesSet = new Set([
-          ...preference.nativeLanguages,
-          languageCode,
-        ])
-
-        if (!checked) nativeLanguagesSet.delete(languageCode)
-        const nativeLanguages: GlobalLanguageCode[] =
-          Array.from(nativeLanguagesSet)
-
-        set(preferenceState, {
-          ...preference,
-          nativeLanguages,
-        })
-      },
-  )
```

Comment:
Why did you delete this? 
It seems like the selection no longer works!

Reply:
Handled in https://github.com/ajktown/wordnote/pull/101/commits/95abf3d4be1171c31f1f52f81ef6f0eadfdca03c